sentry-android-core/build.gradle.kts

```diff
@@ -69,6 +69,16 @@ android {
     lintOptions {
         isWarningsAsErrors = true
         isCheckDependencies = true
+
+        // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks.
+        isCheckReleaseBuilds = false
```

Comment:
I'm not sure I follow: What do you mean by 'vital checks'?
Right now in CI we run `gradle build`, so this task will run all that's needed?

Reply:
text comes from https://android.googlesource.com/platform/tools/base/+/studio-master-dev/build-system/gradle-core/src/main/java/com/android/build/gradle/internal/dsl/LintOptions.java#407
linting should be for debugging purpose and we are gonna fix issues before running a release build.

Reply:
that makes sense.
Just the name of the property is confusing.

Does it mean then: Lint in Release but not in Debug mode?

Reply:
@bruno-garcia it's because the lint task is called Vital
if you run `./gradlew tasks`
> lintVitalRelease - Runs lint on just the fatal issues in the release build.